k8s/yaml/components.yaml

```diff
@@ -2,81 +2,9 @@
 # Source: seldon-core-v2-setup/templates/seldon-v2-components.yaml
```

Comment:
whats the difference between components and runtimes? I see that the runtime is now part of the servers which is slightly confusing to me as I typically change docker images and then redeploy the components. Is this still possible by just applying changes to this file?

Reply:
Images for the core components are defined in the SeldonConfig. SeldonRuntime provides overrides for a few basic aspects and configs at present.